### ISSUE TITLE
Request for skip_batch functionality

### ISSUE THREAD

**Issue by muneebaadil** [thstkdgus35/EDSR-PyTorch#18]
Hi, there was a skip_batch functionality in Lua that skipped the batch for training if the error was above some certain threshold value. Although, it wasn't used frequently in smaller models, but it was extremely useful when training bigger models (such as final version of EDSR); it lead to more stable performance. 

Any chance you'd be adding that?

**Comment by thstkdgus35** [comment]
Hello.

I added a skip_batch function in our training script. You can enable it by setting --skip_threshold [threshold]. Please check it!

Thank you.

**Comment by muneebaadil** [comment]
Apologies if I wasn't able to get the point across. I meant threshold relative to previous batch error, as done [here.](https://github.com/LimBee/NTIRE2017/blob/master/code/opts.lua#L60)

I tried doing it myself but I couldn't find the variable representing the mean loss value of previous batch.

**Comment by thstkdgus35** [comment]
Oh, sorry for misunderstanding.

Actually our first skip_batch function was look like that, so I unconsciously implemented very simple version .

I will include the appropriate version soon.

Thank you.
